bokeh/server/blaze/views.py

```diff
@@ -35,8 +34,8 @@ def _make_range(r):
 @mbsbp.route("/render/<docid>/<datasourceid>/<glyphid>", methods=['GET', 'POST'])
 def render(docid, datasourceid, glyphid):
     #load bokeh document
-    bokehuser = bokeh_app.authentication.current_user()
-    request_username = bokehuser.username
+    #bokehuser = bokeh_app.authentication.current_user()
+    #request_username = bokehuser.username
     clientdoc = bokeh_app.backbone_storage.get_document(docid)
```

Comment:
Things like this, can be removed. Do be aware there are a couple of cases where objects are created for the side effects. These should be preserved and maybe a better pattern found for them in general. The cases I am thinking of are I think when some grid objects are created in the `plotting.py` API. Just creating them with the plot as an argument adds them to the plot, but they are not otherwise used.